pages/404.js

```diff
@@ -0,0 +1,8 @@
+export default function fourOFour() {
+	return (
+		<div className='404--wrapper'>
+			<h1 className='404--oops1'>OOPS!</h1>
```

Comment:
See if you can figure out a way to style this without duplicating the text. If you can't make sure that screen readers read only one.